docker/arm-server/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:22.04
+
+EXPOSE 15550-15560
+EXPOSE 1443
+EXPOSE 6520-6620
+WORKDIR /root
+USER root
+
+RUN set -x
+
+RUN apt update
+RUN apt install -y --no-install-recommends git wget unzip
+RUN apt install -y --no-install-recommends ca-certificates && update-ca-certificates
+RUN git clone https://github.com/google/android-cuttlefish.git
```

Comment:
How about moving this line with line 6 (WORKDIR /root) to before line 19 to separate stages for installing packages and building android-cuttlefish?